docs/source/guides/search/cookbook.rst

```diff
@@ -146,10 +155,13 @@ To get the DOI associated with the bundle the observational product `urn:nasa:pd
 .. code-block:: bash
     :substitutions:
 
-    curl --location --request GET 'http://pds.nasa.gov/api/search/|search_user_guide_api_version|/products/urn:nasa:pds:insight_rad:data_derived:hp3_rad_der_00014_20181211_073042::1.0/bundles?fields=pds:Citation_Information/pds:doi&fields=ops:Identifiers/ops:doi' --header 'Accept: application/kvp+json'
+    curl --get 'https://pds.nasa.gov/api/search/|search_user_guide_api_version|/products/urn:nasa:pds:insight_rad:data_derived:hp3_rad_der_00014_20181211_073042::1.0/bundles' \
+        --data-urlencode 'fields=pds:Citation_Information/pds:doi' \
+        --data-urlencode 'fields=ops:Identifiers/ops:doi' \
```

Comment:
@tloubrieu-jpl as we discussed via a different thread, isn't this not the right way to do it? Shouldn't this be comma separated fields, not resetting it over and over again?